src/components/Staking/StakingCommunityCallout.tsx

```diff
@@ -53,6 +57,7 @@ const StakingCommunityCallout = (props) => {
       alt={translateMessageId("page-staking-image-alt", intl)}
       titleKey={"page-staking-join-community"}
       descriptionKey={"page-staking-join-community-desc"}
+      id={id}
```

Comment:
There is no need to pass down this props since we are already doing `{...props}` above.